<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="5">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_rd">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>√</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_rev">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>1/x</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_module">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>|x|</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_cl">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 127);</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_p">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>π</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_sq">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>x^2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_br1">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_br2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_fact">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>n!</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="pushButton_mod">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>mod</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_two">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>2^x</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_7">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_8">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_9">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="pushButton_plus">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 255, 0);</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_deg">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>x^y</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_5">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_6">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="pushButton_min">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 0);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_ten">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>10^x</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_1">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="pushButton_mult">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 255, 0);</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="pushButton_log">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>log</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pushButton_0">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="pushButton_point">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="pushButton_eq">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 127);</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="pushButton_div">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 255, 0);</string>
      </property>
      <property name="text">
       <string>÷</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
